WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -1109,11 +1098,40 @@ public void onNoteChanged(Note note) {
             }
         }
 
+        // Set 'Reply to (Name)' in comment reply EditText if it's a reasonable size
+        if (!TextUtils.isEmpty(mNote.getCommentAuthorName()) && mNote.getCommentAuthorName().length() < 28) {
+            mEditReply.setHint(String.format(getString(R.string.comment_reply_to_user), mNote.getCommentAuthorName()));
+        }
+
         setComment(localBlogId, note.buildComment());
 
+        // Now we'll add a detail fragment list
+        FragmentManager fragmentManager = getFragmentManager();
+        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
+        mNotificationsDetailListFragment = NotificationsDetailListFragment.newInstance(note.getId());
+        mNotificationsDetailListFragment.setFooterView(mLayoutButtons);
+        // Listen for note changes from the detail list fragment, so we can update the status buttons
+        mNotificationsDetailListFragment.setOnNoteChangeListener(new NotificationsDetailListFragment.OnNoteChangeListener() {
+            @Override
+            public void onNoteChanged(Note note) {
+                mNote = note;
+                mComment = mNote.buildComment();
+                setIdForCommentContainer();
+                updateStatusViews();
+            }
+        });
+        fragmentTransaction.replace(mCommentContentLayout.getId(), mNotificationsDetailListFragment);
+        fragmentTransaction.commitAllowingStateLoss();
+
         getFragmentManager().invalidateOptionsMenu();
     }
 
+    private void setIdForCommentContainer(){
+        if (mComment != null && mCommentContentLayout != null) {
+            mCommentContentLayout.setId((int)mComment.commentID);
```

Comment:
A note on using tags vs IDs, the fragment transaction needs the container `id` where to add the fragment to, and there doesn't seem to be a way to replace the container contents without knowing its `id` in FragmentManager (though this artifact we're making by having a different ID for each _viewpager page's container_ is handled seamlessly by the manager returned by getChildFragmentManager, which is the main purpose of its existence, i.e. handling nested fragments). Setting a tag to find it by tag later then having to set the id anyways seems dumb, so what I did is I'm generating an `id` with a base id plus the Note position index in the notes adapter array to make probability of id collision low (while it is theoretically possible, I don't think it'll ever happen in practice, as we'd have to have a list of more than 65535 items in the notifications list...)

BTW there is also an API to generate a unique id on runtime, but it's also only available since API level 17, which is what we were trying to avoid in the first place 😁 

Commit addressing all this here 8da2cea